src/sound.c

```diff
@@ -108,23 +112,23 @@ delete_sound_callback(soundcb_T *soundcb)
 	}
 }
 
-#if defined(HAVE_CANBERRA) || defined(PROTO)
+# if defined(HAVE_CANBERRA) || defined(PROTO)
 
 /*
  * Sound implementation for Linux/Unix using libcanberra.
  */
-# include <canberra.h>
+#  include <canberra.h>
 
-static ca_context   *context = NULL;
+static ca_context *context = NULL;
 
 // Structure to store info about a sound callback to be invoked soon.
 typedef struct soundcb_queue_S soundcb_queue_T;
 
 struct soundcb_queue_S {
     soundcb_queue_T	*scb_next;
-    uint32_t		scb_id;		// ID of the sound
-    int			scb_result;	// CA_ value
-    soundcb_T		*scb_callback;	// function to call
+    uint32_t		scb_id;         // ID of the sound
+    int			scb_result;     // CA_ value
+    soundcb_T		*scb_callback;  // function to call
```

Comment:
The tabs before the comments are changed to spaces.
Can we keep using tabs here?

Reply:
I think we should get rid of tabs all together and just use spaces

Reply:
This conflicts with the current modeline settings.

As I said somewhere else before, changing tabs to spaces should be a separate patch.

Reply:
Ah yes I remember. You've mentioned in here https://github.com/vim/vim/pull/15789#discussion_r1812794284.

> This conflicts with the current modeline settings.

The modeline should be changed gradually as we format more files

> As I said somewhere else before, changing tabs to spaces should be a separate patch.

Do you mean do a `:retab`? I can do a separate PR and then @dundargoc can rebase on that.

Reply:
The formatter (both clang-fornat and uncrustify) can change tabs to spaces. This should be done via the formatter and not retab.